static/style.css

```diff
@@ -240,6 +240,96 @@ marquee{
             border-radius: 5px;
             background-color:rgb(67, 67, 163);
         }
+
+/* Baby and kids section */
+
+.carousel{
+  width:100%;
+  height:500px;
+  /* left: 400px; */
+
+}
+
+ .carousel-item img{
+  height: 500px;
+  width: 300px;
+   
+ }
+ 
+p{
+  padding: 20px 50px 20px 50px;
+}
```

Comment:
change all tag you have added that is not class or id 
and add class or which is parent of it like

```
p{
  padding: 20px 50px 20px 50px;
}
```


change this to 

```
.offer p{
  padding: 20px 50px 20px 50px;
}
```